larq/models.py

```diff
@@ -239,8 +239,8 @@ def output_pixels(self) -> Optional[int]:
         """Number of pixels for a single feature map (1 for fully connected layers)."""
         if not self.output_shape:
             return None
-        if len(self.output_shape) == 4:
-            return int(np.prod(self.output_shape[1:3]))
+        if len(self.output_shape) == 3 or len(self.output_shape) == 4:
+            return int(np.prod(self.output_shape[1:]))
```

Comment:
In the case of `len(self.output_shape) == 4`, this doesn't giove the same result since:
```python
l = [0, 1, 2, 3]
assert len(l) == 4
assert l[1:3] == [1, 2]
assert l[1:] == [1, 2, 3]
```
Or am I missing something?

Reply:
I see, I locally had `[1:2]` but thought I could combine the two here and make it somewhat cleaner, but indeed this is incorrect :+1:

Reply:
I see, then I guess the following should work:
```suggestion
            return int(np.prod(self.output_shape[1:-1]))
```

Reply:
It should, but I already fixed it by just adding another if-branch returning `shape[1]` for the 1D case, which is probably a bit cleaner than taking the `np.prod` of a one-length array